modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,354 @@
+
+Note: The `max_in_flight: 1` setting is required to preserve message ordering at the partition level. Please refer to xref:components:outputs/kafka_migrator.adoc#max_in_flight[its documentation] for details.
+
+[source,yaml]
+----
+# kafka_migrator_bundle.yaml
+
+input:
+  kafka_migrator_bundle:
+    kafka_migrator:
+      seed_brokers: [ "localhost:9092" ]
+      topics:
+        - '^[^_]' # Skip internal topics which start with `_`
+      regexp_topics: true
+      consumer_group: migrator_bundle
+      start_from_oldest: true
+
+    schema_registry:
+      url: http://localhost:8081
+      include_deleted: true
+      subject_filter: ""
+
+output:
+  kafka_migrator_bundle:
+    kafka_migrator:
+      seed_brokers: [ "localhost:9093" ]
+      max_in_flight: 1
+
+    schema_registry:
+      url: http://localhost:8082
+
+metrics:
+  prometheus: {}
+  mapping: |
+    meta label = if this == "input_kafka_migrator_lag" { "source" }
+----
+
+Now launch the Kafka Migrator Bundle pipeline, and leave it running:
+
+[source,console]
+----
+$ redpanda-connect run kafka_migrator_bundle.yaml
+----
+
+== Check the status of migrated topics
+
+Now you can use the Redpanda https://docs.redpanda.com/current/get-started/rpk[`rpk` CLI tool^] to check which topics and ACLs have been migrated to the `destination` cluster. If you don't have it installed already, please follow the instructions https://docs.redpanda.com/current/get-started/rpk-install[here^].
+
+Note: For now, users need to be migrated manually. However, this step is not required for the current demo. Similarly, roles are specific to Redpanda and, for now, will also require manual migration if the `source` cluster is based on Redpanda.
+
+[source,console]
+----
+$ rpk -X brokers=localhost:9093 -X admin.hosts=localhost:9645 topic list
+NAME      PARTITIONS  REPLICAS
+_schemas  1           1
+bar       2           1
+foo       2           1
+
+$ rpk -X brokers=localhost:9093 -X admin.hosts=localhost:9645 security acl list
+PRINCIPAL      HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+User:redpanda  *     TOPIC          bar            LITERAL                READ       DENY
+User:redpanda  *     TOPIC          foo            LITERAL                READ       ALLOW
+----
+
+== Check metrics to monitor progress
+
+Redpanda Connect provides a comprehensive suite of metrics in various formats, which you can use to monitor its performance in your observability stack. Besides the xref:components:metrics/about.adoc#metric-names[standard Redpanda Connect metrics], the `kafka_migrator` input also emits an `input_kafka_migrator_lag` metric for monitoring the migration progress of each topic and partition (for example in Prometheus format).
```

Comment:
```suggestion
Redpanda Connect provides a comprehensive suite of metrics in various formats, which you can use to monitor its performance in your observability stack. Besides the xref:components:metrics/about.adoc#metric-names[standard Redpanda Connect metrics], the `kafka_migrator` input also emits an `input_kafka_migrator_lag` metric for monitoring the migration progress of each topic and partition. For example, in Prometheus format.
```

Reply:
I reworded this in a different way, hope that's OK.